overlord/servicestate/quota_handlers_test.go

```diff
@@ -868,6 +868,60 @@ func (s *quotaHandlersSuite) TestQuotaRemove(c *C) {
 	checkSvcAndSliceState(c, "test-snap.svc1", "foo", 0)
 }
 
+func (s *quotaHandlersSuite) TestQuotaSnapSubgroupUnmixable(c *C) {
```

Comment:
we also need a unit test for an existing quota group with mixed snaps and groups where we try to add a new resource limit to it (or increase the memory limit will do for now I think)

Reply:
Really good point, i'll add this